README.md

```diff
@@ -121,6 +121,9 @@ Here is what you need to be able to run Cal.
    ```sh
    git clone https://github.com/calcom/cal.com.git
    ```
+   > If you are on windows,
+   ```git clone -c core.symlinks=true https://github.com/calcom/cal.com.git``` <br>
```

Comment:
This would still not work unless it's from a command shell with elevated privileges (Admin). Probably a good idea to add this information too.

Reply:
ah right, added.